botnetdobbs/Laravel-Tdd
# implement auth and permissions

[Finishes]

#### What does this PR do?
- this PR implements controls based on the Laravel's auth system e.g a user cannot create post if not logged in.
#### Description of Task to be completed?
- write tests for inbuilt Laravel auth system
- implement permission controls based on auth
- update previous tests
#### How should this be manually tested?
- clone the repo then checkout to this branch `ft-create-new-post`
- run composer install
- create a database and put in your creds in the .env
- create a sqlite db file for testing `touch database/testing.sqlite`
- run `./vendor/bin/phpunit`
- create an account `/register` and proceed to login if not automatically logged in `/login`
- go to `/posts/create`
- Should be redirected to the `/posts` page
#### Any background context you want to provide?
N/A
#### What are the relevant stories?
[]()
#### Screenshots (if appropriate)
